homeassistant/components/onvif/camera.py

```diff
@@ -79,9 +89,13 @@
 SERVICE_PTZ_SCHEMA = vol.Schema(
     {
         ATTR_ENTITY_ID: cv.entity_ids,
-        ATTR_PAN: vol.In([DIR_LEFT, DIR_RIGHT, PTZ_NONE]),
-        ATTR_TILT: vol.In([DIR_UP, DIR_DOWN, PTZ_NONE]),
-        ATTR_ZOOM: vol.In([ZOOM_OUT, ZOOM_IN, PTZ_NONE]),
+        vol.Optional(ATTR_PAN): vol.In([DIR_LEFT, DIR_RIGHT, None]),
```

Comment:
We don't want to accept `None` as a valid value.
```suggestion
        vol.Optional(ATTR_PAN): vol.In([DIR_LEFT, DIR_RIGHT]),
```